exonum/src/node/mod.rs

```diff
@@ -983,10 +990,18 @@ pub fn create_private_api_handler(
     blockchain: Blockchain,
     shared_api_state: SharedNodeState,
     api_sender: ApiSender,
+    config: &NodeApiConfig,
```

Comment:
Why not passing only `enable_unsafe_api`?

Reply:
IMHO pass api config to the function that creates api is better than pass one concrete flag:
1. It's more clear to the callee: "Hm, I want to create a private api, so I should provide api config" instead of "Hm, I should provide some flag from the config.. Which one?".
2. Caller provides config, and callee deals with it as it should. There is no risk that somehow a random boolean value will be passed.
3. It's a reference, so we shouldn't worry about performance issues.